.travis.yml

```diff
@@ -24,15 +24,19 @@ matrix:
 
 script:
   - cd $TRAVIS_BUILD_DIR/simulator
-  - make mipt-mips
+#  - make mipt-mips
+  - mkdir $TRAVIS_BUILD_DIR/simulator/build && cd $TRAVIS_BUILD_DIR/simulator/build
+  - cmake ./..
+  - make
   - cd $TRAVIS_BUILD_DIR/traces
   - make MIPS_AS=$TRAVIS_BUILD_DIR/binutils/cross/mips/bin/mipsel-unknown-linux-gnu-as MIPS_LD=$TRAVIS_BUILD_DIR/binutils/cross/mips/bin/mipsel-unknown-linux-gnu-ld
-  - cd $TRAVIS_BUILD_DIR/simulator
-  - make test
+  - cd $TRAVIS_BUILD_DIR/simulator/build
+  - ctest
 # Run Clang-Tidy
-  - if [ $CXX == clang++-5.0 ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then make TIDY=clang-tidy-5.0 tidy-check; fi
+#  - if [ $CXX == clang++-5.0 ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then make TIDY=clang-tidy-5.0 tidy-check; fi
```

Comment:
Please return condition back, as there is no sense to run Clang-Tidy in GCC/MacOS build